rest-api/core-rest/src/main/webapp/lib/gaffer.js

```diff
@@ -188,6 +188,15 @@ function updateTitle(properties) {
     });
 }
 
+function updateFooter(properties) {
+    updateElementWithId("swagger-ui-container", "gaffer.version", properties, function(value, id) {
+        var footerText = $('#' + id).find("div.footer").text();
```

Comment:
You could store $('#' + id).find("div.footer") as a variable so you don't have to search for it twice.

Reply:
Im not sure if this will be beneficial.  I think it makes it harder to read without making any improvement.